### ISSUE TITLE
Issues with panther_transfer_on_finish

### ISSUE THREAD

**Issue by hjia1005** [usgs/pestpp#285]
I am running pestpp-swp, after each run, over 10 files should be transferred to the master folder. Currently I am running only a few realizations to test the workflow and the file transfer. However, it looks like as soon as the last model run is done, pestpp-swp is terminated before all files from the last model are properly transferred. No matter how many workers I use, I end up with insufficient files transferred from the last model run but I can get all files from the previous models. Anyone has encountered the same issue?

**Comment by jtwhite79** [comment]
The way the file transfers work is the worker initiates the transfer so I wonder if the master sees that all requested transfers are complete and returns control to the pestpp-swp before the last/slower worker can tell the master it has more files to transfer.  Can you email me your rmr file and the two worker agent .rec file so I can try to piece together what is happening?

**Comment by hjia1005** [comment]
Thanks Jeremy. I ran the full realizations yesterday and was prepared to sacrifice the results from the last model run. It has finished right now, when I checked the results, surprisingly every file has been successfully transferred to the master folder. I can't explain why. When I ran 2 or 3 realizations, the issue mentioned in my previous post kept happening. But I guess it's solved now and I will close the issue. Thank you very much.
